apps/web-api/src/admin/member.controller.ts

```diff
@@ -27,6 +27,17 @@ export class MemberController {
     return await this.memberService.findMemberByAccessLevels(query);
   }
 
+  /**
+   * Returns a single member by uid.
+   * Used by Back Office to refresh roles and member data.
+   */
+  @Get(':uid')
+  @UseGuards(AdminAuthGuard)
```

Comment:
Looks like you have to use `DemoDayAdminAuthGuard` here to be able for `DEMO_DAY_ADMIN` roles